bot/cogs/bubble_wrap_cog.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import discord
+import discord.ext.commands as commands
+
+log = logging.getLogger(__name__)
+
+class RandomStuffCog(commands.Cog):
```

Comment:
Lets rename the cog and file to MemesCog and memes_cog.py respectively, we have a few other meme commands that are being added and this is a good time to set up that grouping